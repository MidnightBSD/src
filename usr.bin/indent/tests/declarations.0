/* $FreeBSD: stable/11/usr.bin/indent/tests/declarations.0 313544 2017-02-10 09:31:39Z pstef $ */
/* See r303570 */
void t(void) {
	int a,
	b,
	c;
	int
	*d,
	*e,
	*f;
	int (*g)(),
	(*h)(),
	(*i)();
	int j,
	k,
	l;
	int m
	,n
	,o
	;
}
