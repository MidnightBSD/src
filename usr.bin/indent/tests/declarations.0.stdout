/* $FreeBSD$ */
/* See r303570 */
void 
t(void)
{
	int		a, b, c;
	int
		       *d, *e, *f;
	int 		(*g) (), (*h) (), (*i) ();
	int		j, k, l;
	int		m
		       ,n
		       ,o
		       ;
}
